modules/features/manatime_feature_timesheets/plugins/content_types/timesheet_view.inc

```diff
@@ -23,11 +23,11 @@ $plugin = array(
 function timesheet_view_render($subtype, $conf, $panel_args, $context) {
   if (isset($context[0]->data)) {
     // Get the records of this timesheet.
-    $timesheet = new Timesheet($context[0]->data->nid);
+    $node = node_load($context[0]->data->nid);
```

Comment:
I'm almost sure $context[0]->data is the real node. Take a look, if yes, you don't need to load the node again.